src/components/tile-select/tile-select.test.js

```diff
@@ -0,0 +1,641 @@
+          <TileSelectComponent className={className} />
+        );
+        tileSelectDataComponent().should("have.class", className);
+      }
+    );
+
+    it("should check footer for TileSelect component", () => {
+      CypressMountWithProviders(
+        <TileSelectComponent
+          footer={
+            <Button
+              ml={1}
+              buttonType="tertiary"
+              iconPosition="after"
+              iconType="home"
+            >
+              Footer Button
+            </Button>
+          }
+        />
+      );
+      getComponent("button")
+        .should("be.visible")
+        .and("have.text", "Footer Button");
+    });
+
+    it.each([CHARACTERS.DIACRITICS, CHARACTERS.SPECIALCHARACTERS])(
+      "should check prefixAdornment as %s for TileSelect component",
+      (PrefixAdornment) => {
+        CypressMountWithProviders(
+          <TileSelectComponent
+            prefixAdornment={
+              <>
+                <Pill fill mr={1} mb="4px">
+                  {PrefixAdornment}
+                </Pill>{" "}
+              </>
+            }
+          />
+        );
+        getComponent("pill")
+          .should("have.text", PrefixAdornment)
+          .and("be.visible");
+      }
+    );
+
+    it("should check additionalInformation for TileSelect component", () => {
+      CypressMountWithProviders(
+        <TileSelectComponent
+          additionalInformation={
+            <>
+              <Pill fill mr={1} mb="4px">
+                Further information
+              </Pill>
+            </>
+          }
+        />
+      );
+      getComponent("pill")
+        .should("be.visible")
+        .and("have.text", "Further information");
+    });
+  });
+
+  describe("check props for MultiTileSelect component", () => {
+    it.each(["radio", "checkbox"])(
+      "should check type as %s for TileSelect component",
+      (type) => {
+        CypressMountWithProviders(
+          <MultiTileSelectComponent multiSelect type={type} />
+        );
+        inputElement().should("have.attr", "type", type);
+      }
+    );
+
+    it("should check multiSelect for TileSelect component", () => {
+      CypressMountWithProviders(<MultiTileSelectGroupComponent multiSelect />);
+      inputElement().eq(0).click().and("have.attr", "aria-checked", "true");
+
+      inputElement().eq(1).click().and("have.attr", "aria-checked", "true");
+
+      inputElement().eq(2).and("have.attr", "aria-checked", "false");
+
+      inputElement().eq(3).and("have.attr", "aria-checked", "false");
+    });
+  });
+
+  describe("check props for Accordion", () => {
+    it.each([
+      [false, "0px"],
+      [true, "152px"],
+    ])(
+      "should check when accordionExpanded is set as %s for TileSelect component",
+      (boolV, maxHeight) => {
+        CypressMountWithProviders(
+          <AccordionTileSelectComponent accordionExpanded={boolV} />
+        );
+        getElement("tile-select-accordion-content")
```

Comment:
can we check if the children (content) is visible? instead of checking the height of component?